### ISSUE TITLE
Miniature graph in slack alerts is not consistent with dashboard

### ISSUE THREAD

**Issue by josebolos** [SpeedCurve-Metrics/SpeedCurve#175]
The miniature graph shown in the Slack alerts has a green band marking the performance budget.

![screen shot 2017-10-30 at 10 54 55-2](https://user-images.githubusercontent.com/1792637/32169494-a22cdfc4-bd68-11e7-9804-2120ccf504a8.png)

This is different from how limits are displayed in the dashboard, with a thing red line, with the parts of the graph above the budget displayed also in red.

![screen shot 2017-10-30 at 10 55 17](https://user-images.githubusercontent.com/1792637/32169499-a75a40f4-bd68-11e7-900b-f1ce127c48d9.png)

This is proving confusing for some of my colleagues receiving the alerts. Some of them think that the limit is in the centre of the green band and there's some kind of "error margin" around the budget, when the limit is actually the top of the green band. All of the graph is yellow in the miniature, which makes it hard to identify if a particular point is above the budget or not. For example, the last test displayed in the miniature above.

I think it would be less confusing if limits were displayed in the same way in both places.

**Comment by josephwynn-sc** [comment]
Hi there!

SpeedCurve budgets have an **absolute** threshold, which you see represented as the red line on your second chart. They can also have a **relative** threshold (referred to as "rate of change" in the budget settings):

![screenshot from 2017-10-31 08-03-50](https://user-images.githubusercontent.com/31634862/32190273-1b665308-be12-11e7-9d39-ed48e4598295.png)

Your first chart is from a rate of change alert -- the latest value is more than 10% different to the previous value. The green band on this chart shows the _average_ boundary for the rate of change.

Hopefully that helps to clear things up. We're always open to suggestions if you have any ideas for ways to clarify the meaning of these charts!

**Comment by josebolos** [comment]
Hi @josephwynn-sc 

I stand corrected, all the alerts that we were getting were for relative limits so I didn't realise that the graph for the absolute ones was indeed the same.

I've been trying to find in the documentation a place where it explains what the green band is but I've been unsuccessful so far. The receivers of the alerts are not web performance engineers, and they are not familiar with the tool. Having things unlabelled makes things confusing for us.

Is there any docs that I can send to my colleagues explaining the meaning of the green band? Alternatively, is there any chance that some kind of legend or help text can be added explaining its meaning?

Thanks!

**Comment by josephwynn-sc** [comment]
I've created a new support article that documents some of what we've talked about in this issue: http://support.speedcurve.com/synthetic-settings/interpreting-performance-budget-notifications

Performance budgets are one of the areas that we're looking to revamp at some point. I've made a note of this issue so that we can take it into account when thinking about how best to visualise the thresholds.

By the way - thanks for the awesome feedback!
